modules/programs/git.nix

```diff
@@ -61,7 +61,7 @@ let
   signModule = types.submodule {
     options = {
       key = mkOption {
-        type = types.str;
+        type = types.nullOr types.str;
         description = "The default GPG signing key fingerprint.";
```

Comment:
```suggestion
        description = ''
          The default GPG signing key fingerprint.
          Set to <literal>null</literal> to disable using a default key for signing.
        '';
```

What do you think?

Reply:
A also wanted to update the description but couldn’t come up with anything sensible. The problem here is that you are saying “turn off default key to use default key”, I think for some this might be confusing. This needs some rephrasing, to explicitly highlight that by using `null` the signing key would depend on commit’s author.

Reply:
Maybe this?
> Set to `null` to let GPG decide what signing key to use depending on commit’s author.

Reply:
Sounds goood

Reply:
@berbiche I’ve updated the description. Resolving this.